lib/hooks/docker-hub-automated-build/help.html.haml

```diff
@@ -0,0 +1,4 @@
+  %dl
```

Comment:
This indentation seems causing CI failure (`Haml::SyntaxError: Indenting at the beginning of the document is illegal.`). Would you please remove this?